### PR TITLE
Update rubygem-mini_portile2 to 2.5.1

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
[test rpm]

File: packages/foreman/rubygem-mini_portile2/rubygem-mini_portile2.spec [hunk 1]
```diff
@@ -71,12 +70,10 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%{gem_instdir}/.concourse.yml
+%{gem_instdir}/.github
```
Comment (ekohl) [inline packages/foreman/rubygem-mini_portile2/rubygem-mini_portile2.spec:73]: I think this should be excluded.

File: packages/foreman/rubygem-mini_portile2/rubygem-mini_portile2.spec [hunk 2]
```diff
@@ -71,12 +70,10 @@ cp -a .%{gem_dir}/* \
...
 %exclude %{gem_instdir}/.gitignore
-%exclude %{gem_instdir}/.travis.yml
 %license %{gem_instdir}/LICENSE.txt
-%{gem_instdir}/appveyor.yml
-%{gem_instdir}/concourse
+%{gem_instdir}/SECURITY.md
```
Comment (ekohl) [inline packages/foreman/rubygem-mini_portile2/rubygem-mini_portile2.spec:76]:
```suggestion
%doc %{gem_instdir}/SECURITY.md
```

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by ehelms** [comment]
Addresses comments and adds a commit to update gem2rpm scl template

**Comment by ehelms** [comment]
Note once this goes in the current nokogiri dependencies will be broken, so this will need to be coordinated with https://github.com/theforeman/foreman-packaging/pull/6583 such that they are merged close together. My testing locally with mock indicates the nokogiri PR should build once this and https://github.com/theforeman/foreman-packaging/pull/6581 are present.

**Comment by ehelms** [comment]
This will need to be reviewed and timed with https://github.com/theforeman/foreman-packaging/pull/6583 also being approved. The rubygem-racc requirement for nokogiri has been merged and built.

**Review by ekohl — APPROVED** [review]

**Pull request merged**